Green-space/clean.js

```diff
@@ -0,0 +1,68 @@
+"use strict";
+const turf = require("@turf/turf");
+const multipolygon = require("turf-multipolygon");
+const fs = require("fs");
+const geojson2osm = require('geojson2osm');
+const reader = require('geojson-writer').reader
+
+const collection1 = turf.featureCollection([]);
+
+let pools = reader('parks-source.json')
+
+pools.features.map(result => {
```

Comment:
Using `forEach()` would be more appropriate for this operation since `map()` it continuously builds an Array in memory.

Reply:
Didn't I borrow this from your code? :)

Reply:
Probably! You shouldn't use bad references 🤣